_includes/manuals/1.13/3.2.3_installation_scenarios.md

```diff
@@ -79,7 +79,8 @@ foreman-installer \
   --puppet-server-storeconfigs-backend=puppetdb
 {% endhighlight %}
 
-Be aware that foreman-installer does not setup the PuppetDB server itself.
+Be aware that foreman-installer does not setup the PuppetDB server itself. Starting with 1.13 of foreman-installer, only
+setups using PuppetLabs Puppet 4 AIO packages are supported for PuppetDB integration using these parameters..
```

Comment:
Extra full stop at the end of the sentence should be removed, and Puppet Labs should be two words (or even "Puppet" now it's renamed).